test/e2e/network/dns_common.go

```diff
@@ -494,7 +494,7 @@ func createProbeCommand(namesToResolve []string, hostEntries []string, ptrLookup
 // createTargetedProbeCommand returns a command line that performs a DNS lookup for a specific record type
 func createTargetedProbeCommand(nameToResolve string, lookup string, fileNamePrefix string) (string, string) {
 	fileName := fmt.Sprintf("%s_udp@%s", fileNamePrefix, nameToResolve)
-	probeCmd := fmt.Sprintf("dig +short +tries=12 %s %s > /results/%s", nameToResolve, lookup, fileName)
+	probeCmd := fmt.Sprintf("for i in `seq 1 150`; do dig +short +tries=12 %s %s > /results/%s; sleep 1; done", nameToResolve, lookup, fileName)
```

Comment:
That's going to add 2.5 minutes to a failing test.  Under what conditions do we think that success after 2.4 minutes is acceptable?

I might be ok to bump this to 10 seconds, maaaaaaaaybe 30 seconds.  We could probably also remove the "tries" argument, since we'll retry in a loop.

Reply:
Not really. All tests wait a maximum of 10 minutes per test (https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/dns_common.go#L508) while waiting for all the desired files for their respective DNS entries to be created by the containers. So, the overall execution time is not affected in case of failure.

Reply:
nonetheless, I've updated the commit with your suggestions.